src/webapp/components/modal-selector/ModalSelector.tsx

```diff
@@ -29,20 +30,20 @@ const _ModalSelector = React.memo((props: ModalSelectorProps) => {
         });
     }, [items, search]);
 
-    const Row = ({ index, style }: { index: number; style: React.CSSProperties }) => {
-        const item = itemsToRender[index];
-        if (!item) return null;
+    // const Row = ({ index, style }: { index: number; style: React.CSSProperties }) => {
+    //     const item = itemsToRender[index];
+    //     if (!item) return null;
 
-        return (
-            <div style={{ ...style, width: "initial" }}>
-                <Grid item xs={12}>
-                    <Button onClick={() => onSelectItem(item)} color="primary" disableElevation>
-                        {item.text}
-                    </Button>
-                </Grid>
-            </div>
-        );
-    };
+    //     return (
+    //         <div style={{ ...style, width: "initial" }}>
+    //             <Grid item xs={12}>
+    //                 <Button onClick={() => onSelectItem(item)} color="primary" disableElevation>
+    //                     {item.text}
+    //                 </Button>
+    //             </Grid>
+    //         </div>
+    //     );
+    // };
```

Comment:
Do we need to keep these comments? If yes, add a comment to the code explaining why.